ros1_ign_bridge/include/ros1_ign_bridge/factory.hpp

```diff
@@ -114,7 +115,7 @@ class Factory : public FactoryInterface
     const boost::shared_ptr<ros::M_string> & connection_header =
```

Comment:
I was looking at this function on `ros1_bridge` yesterday and noticed they [print more](https://github.com/ros2/ros1_bridge/blob/master/include/ros1_bridge/factory.hpp#L148-L188) debug info than we do. We could do the same here, using the type names:

Reply:
90ce9939f7c0218290f7c21fd20a0971a845cbaa